### PR TITLE
Add guide on how to get best streams

### PR DESCRIPTION
**Pull request by qtchaos**
Closes https://github.com/movie-web/movie-web/issues/758.

* Adds a new "Extra" category meant to be used for extra topics, so that we can link to them in the Discord, etc.
* Wrote a small section for all the current supported ways to proxy requests (+ extension) so that users have a bit more context.
* Removes the `Edit page on GitHub` button, I felt as it got in the way more than it helped and incentivizes random docs changes rather than proper PRs with approved issues.

### CODE REVIEW

File: content/5.extra/1.streaming.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Streaming
+
+This page explains all the different ways you can enable movie-web to stream your favorite movies & TV shows, each with their own pros and cons.
+
+<!-- ## Method 1 - Browser extension
+
+The movie-web browser extension is the easiest way to be able to watch media with fast streaming speeds, it is available for both [Chrome] and [Firefox].
+
+This method is great if you only use movie-web on your computer. If you use a mobile device or smart TV, you'll unfortunately have to stick to using a proxy since these devices don't usually support browser extensions.
```
Comment (JorrinKievit) [inline content/5.extra/1.streaming.md:13]: Firefox mobile (only for android) will also work (hopefully) https://addons.mozilla.org/en-US/android/

Comment (qtchaos) [reply content/5.extra/1.streaming.md:13]: Ah yeah, thought about that but wasn't sure if that was something that will be supported or not :)

File: content/5.extra/1.streaming.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Method 1 - Self-hosted proxy
+
+Self-hosting a proxy is an easy way to get good streaming speeds, [we have a guide](../2.proxy/1.deploy.md) that explains how to set up one for **free**.
+
+This method is recommended if you want to host a proxy for your friends and or family to use, as it is the fastest and most reliable way to stream media at the moment.
+
+<!-- This method is recommended if you want to host a proxy for your friends and family to use, or if you want to use movie-web on a device that doesn't support the [browser extension](#method-1---browser-extension), such as a smart TV or mobile device. -->
+
+Once you follow the guide on how to set up a proxy, you can use the URL of your proxy in the client either by using the [`VITE_CORS_PROXY_URL`](../3.client/3.configuration.md#vite_cors_proxy_url) variable if you're self-hosting or if you're using the main website, by going into the `Settings`, then scrolling all the way down to `Connections` and enabling the `Use custom proxy workers` option and entering your URL from earlier.
```
Comment (mrjvs) [inline content/5.extra/1.streaming.md:25]: I like it, but I think it should contain a list of steps to get the self-hosted proxy in the right place.
The goal of the page is to inform the user of the best ways to get streams, and how to actually get it setup.
So if somehow whines about having buffering, we can link this page and they wouldn't need any extra help.

Something like:

```md
...Selfhost proxy gives access to faster streams than default..

...To deploy follow this steps:...
1. Setup a proxy on netlify: ../2.proxy/1.deploy.md
2. Go to settings page on movie-web (link to it)
3. Enabled "custom proxy urls" if its not already
4. Click "add new url" if there isn't one (show screenshot of completed settings page)
5. Copy netlify URL (show image on how to)

::alert
...If you're selfhosting, you can input this URL in VITE_CORS_PROXY_URL.
::
```

File: content/5.extra/1.streaming.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Method 2 - Public proxy
+
+The public proxy is the easiest way to get started with movie-web as it is the default, it is hosted by us and is completely free to use.
+
+If you are using the main website, then you are most likely already using the public proxy. This is not the case with self-hosted clients, there is no proxy set up by default on self-hosted clients and you will need to [set up one yourself](../2.proxy/1.deploy.md).
```
Comment (mrjvs) [inline content/5.extra/1.streaming.md:31]: We should probably mention the cons of using the public proxy.
It has slower streams and buffers quite often.

**Review by mrjvs — CHANGES_REQUESTED** [review]

**Comment by qtchaos** [comment]
Should probably link back to the Streaming guide from the proxy deploy guides so that it  actually shows how to set up the client.

**Review by mrjvs — APPROVED** [review]

**Pull request merged**